features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/ImageWireframeHelper.kt

```diff
@@ -65,7 +76,179 @@ internal class ImageWireframeHelper(
         return imageWireframe
     }
 
-    private companion object {
-        private const val DRAWABLE_CHILD_NAME = "drawable"
+    @Suppress("NestedBlockDepth")
+    internal fun createCompoundDrawableWireframes(
+        view: TextView,
+        mappingContext: MappingContext,
+        startIndex: Int
+    ): MutableList<MobileSegment.Wireframe> {
+        val result = mutableListOf<MobileSegment.Wireframe>()
+        var currentIndex = startIndex
+
+        view.compoundDrawables.indices.forEach {
+            val drawable = view.compoundDrawables[it]
+            val density = mappingContext.systemInformation.screenDensity
+
+            if (drawable != null) {
+                val compoundDrawableLocationMap = createCompoundDrawableLocationMap(
+                    view = view,
+                    drawable = drawable,
+                    density = density,
+                    index = it
+                )
+
+                val drawableX = compoundDrawableLocationMap[it]?.first
+                val drawableY = compoundDrawableLocationMap[it]?.second
```

Comment:
I think you can use `android.graphics.Point` class in order to avoid using `Pair`.